objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/ObjectRecognitionConfigHandler.java

```diff
@@ -72,31 +72,31 @@
-    private static final String CAMERA = "camera";
-    private static final String NETWORK = "network";
-    private static final String FTP = "ftp";
-    private static final String NEURAL_NET = "neuralNet";
-    private static final String YOLO = "yolo";
-    private static final String NONE = "none";
-    private static final String DEFAULT = "default";
-    private static final String TEST = "test";
-    private static final String POLL_TIME = "pollTime";
-    private static final String POLL_RATE = "pollRate";
-    private static final String ALLOW_QUERIES = "allowQueries";
-    private static final String MAX_RUNNING_THREADS = "maxRunningThreads";
-    private static final String MAX_QUEUED_TASKS = "maxQueuedTasks";
-    private static final String SUPPRESS_EMPTY_NEURAL_NET_RESULTS = "suppressEmptyNeuralNetResults";
-    
+    private static final String CONFIG                              = "config";
+    private static final String OBJ_REC_CONFIG                      = "objRecConfig";
+    private static final String GENERAL                             = "general";
+    private static final String DATA_SOURCE                         = "dataSource";
+    private static final String FILE                                = "file";
+    private static final String CAMERA                              = "camera";
+    private static final String NETWORK                             = "network";
+    private static final String FTP                                 = "ftp";
+    private static final String NEURAL_NET                          = "neuralNet";
+    private static final String YOLO                                = "yolo";
+    private static final String NONE                                = "none";
+    private static final String DEFAULT                             = "default";
+    private static final String TEST                                = "test";
+    private static final String POLL_TIME                           = "pollTime";
+    private static final String POLL_RATE                           = "pollRate";
+    private static final String ALLOW_QUERIES                       = "allowQueries";
+    private static final String MAX_RUNNING_THREADS                 = "maxRunningThreads";
+    private static final String MAX_QUEUED_TASKS                    = "maxQueuedTasks";
+    private static final String SUPPRESS_EMPTY_NEURAL_NET_RESULTS   = "suppressEmptyNeuralNetResults";
+
     // Constants for Query Parameters
-    private static final String OPERATION = "operation";
-    private static final String PROCESS_NEXT_FRAME = "processnextframe";
-    private static final String UPLOAD = "upload";
-    private static final String DELETE = "delete";
+    private static final String OPERATION           = "operation";
+    private static final String PROCESS_NEXT_FRAME  = "processnextframe";
+    private static final String UPLOAD              = "upload";
+    private static final String DELETE              = "delete";
```

Comment:
Usually we try to filter out these _my IDE decided to reformat things just for fun_ types of changes.  Not a big deal, but you might want to unset whatever it is that decides to line things up halfway across the room.

I don't think I saw any substantive changes in here.

Reply:
I'll unset those now.